recipes/ruff-lsp/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "ruff-lsp" %}
+{% set version = "0.0.42" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/ruff_lsp-{{ version }}.tar.gz
+  sha256: 249581bec445ccdffb313f8e6ef8ce20d14b8bba36a882a9d041c8334af68462
+
+build:
+  entry_points:
+    - ruff-lsp = ruff_lsp.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - hatchling
+    - pip
+  run:
+    - python >=3.7
+    - packaging >=23.1
+    - pygls >=1.1.0
+    - lsprotocol >=2023.0.0a1
+    - ruff >=0.0.274
+    - typing-extensions
+
+test:
+  imports:
+    - ruff_lsp
+  commands:
+    - pip check
+    - ruff-lsp --help
+  requires:
+    - pip
+
+about:
+  summary: A Language Server Protocol implementation for Ruff.
+  license: MIT
+  license_file: LICENSE
```

The linter is complaining that this is missing a required `home` key - e.g.
https://github.com/conda-forge/staged-recipes/blob/7b820acf2a70a2b92040b4d7929aff7df5f0b9aa/recipes/example/meta.yaml#L66-L67
<br>
Not sure why `grayskull` didn't add that, but the below should do the trick:
```suggestion
about:
  home: https://github.com/astral-sh/ruff-lsp
  summary: A Language Server Protocol implementation for Ruff.
  license: MIT
  license_file: LICENSE
```